.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/google-cloud-ruby/issues
+  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+ruby
+
+If you are still having issues, please be sure to include as much information as possible:
+
+#### Environment details
+
+  - OS:
+  - Ruby version:
+  - Gem name and version:
+
+#### Steps to reproduce
+
+  1. ...
+
+#### Code example
+
+```ruby
+# example
+```
+
```

Comment:
Can we add something like:
```
#### Backtrace / expected output

```sh
$ irb
irb(main):001:0> foo
NameError: undefined local variable or method `foo' for main:Object
	from (irb):1
	from .rbenv/versions/2.4.2/bin/irb:11:in `<main>'
```
```

Reply:
👋 Feel free to submit follow up PRs that mold this to your liking :) The important part for me was the consistent language around Support requests, contracts, etc.

Reply:
kk, will do